WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -19,6 +21,16 @@ final class PaymentCaptureOrchestrator {
             onCompletion(.failure(CardReaderServiceError.paymentCapture()))
             return
         }
+
+        if !PKPassLibrary.isSuppressingAutomaticPassPresentation() {
+            PKPassLibrary.requestAutomaticPassPresentationSuppression() { result in
+                guard result == .success else {
+                    DDLogWarn("Automatic Pass (Apple Pay) Presentation Suppression failed. Reason: \(result.rawValue)")
```

Comment:
Just got here from the p2 where the entitlement was approved. I know this is not yet ready for review but do you think we should also log this to sentry? It seems like an error we would want to fix sooner than later.

Reply:
Yes, lets

Reply:
Added sentry in https://github.com/woocommerce/woocommerce-ios/pull/4218/commits/784c6cb0cefd01a19e1aa6665d837b985f499c22